### DISCUSSION TITLE
Setting up translator services

### DISCUSSION THREAD

**Discussion by doceoriver** [AcclaroInc/craft-translations#486] [Q&A] [answered]
Hi! The [Configuring Translator Services](https://github.com/AcclaroInc/craft-translations/wiki/Configuring-Translator-Services) page doesn't work and I have a client using this plugin wanting to integrate it with my memoQ server. Is it possible to do this via API?

**Comment by sidedwards** [comment, marked as answer]
Hi, thanks for reaching out. We don't currently have a direct memoq integration. My recommendation would be to set up a local translator service and use the import/export functionality, import the files to memoq, translate, and upload them back into Craft.

**Reply by doceoriver** [reply]
Ok, thanks!
